src/spec/junit_formatter.cr

```diff
@@ -67,6 +67,8 @@ module Spec
       HTML.escape(classname(result), io)
       io << %(" name=")
       HTML.escape(escape_xml_attr(result.description), io)
+      io << %(" line=")
+      HTML.escape(result.line.to_s, io)
```

Comment:
Escaping is okay (better safe than sorry), but not really necessary here. `result.line` is an `Int32`, so its stringification cannot include any characters that need to be escaped in HTTP.